From bfc215ae39926c05fb95230bac5b6915bd090a5b Mon Sep 17 00:00:00 2001
From: Victoria Milhoan <vicki.milhoan@freescale.com>
Date: Fri, 14 Nov 2014 13:31:29 -0700
Subject: [PATCH 1538/1594] MLK-9769-12 caam secvio - use explicit includes
 for OF header files

commit 24a6bce12b6361119874064e3a3a9d4a99d32513 from
git://git.freescale.com/imx/linux-2.6-imx.git

Add explicit includes for OF header files.  This patch is based on
upstream commit 5af5073004071cedd0343eee51d77955037ec6f3.

Signed-off-by: Victoria Milhoan <vicki.milhoan@freescale.com>
---
 drivers/crypto/caam/secvio.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/crypto/caam/secvio.c b/drivers/crypto/caam/secvio.c
index 1c94560..3ea3636 100644
--- a/drivers/crypto/caam/secvio.c
+++ b/drivers/crypto/caam/secvio.c
@@ -4,6 +4,8 @@
  * Copyright (C) 2012-2015 Freescale Semiconductor, Inc., All Rights Reserved
  */
 
+#include <linux/of_address.h>
+#include <linux/of_irq.h>
 #include "compat.h"
 #include "intern.h"
 #include "secvio.h"
-- 
1.7.5.4

